api/src/test/java/org/pmiops/workbench/featuredworkspace/FeaturedWorkspaceTest.java

```diff
@@ -41,23 +40,26 @@
 public class FeaturedWorkspaceTest {
 
   @MockBean private FeaturedWorkspaceDao mockFeaturedWorkspaceDao;
-  @MockBean private FeaturedWorkspaceMapper mockFeaturedWorkspaceMapper;
   @MockBean private FireCloudService mockFireCloudService;
-  @MockBean private InitialCreditsExpirationService mockInitialCreditsExpirationService;
-  @MockBean private WorkspaceMapper mockWorkspaceMapper;
 
   @Autowired private FeaturedWorkspaceService featuredWorkspaceService;
 
   private DbWorkspace dbWorkspace;
+  private List<RawlsWorkspaceListResponse> rawlsWorkspaces;
 
   @TestConfiguration
   @Import({
+    CommonMappers.class,
     FakeClockConfiguration.class,
+    FeaturedWorkspaceMapperImpl.class,
     FeaturedWorkspaceServiceImpl.class,
+    FirecloudMapperImpl.class,
+    WorkspaceMapperImpl.class,
   })
   @MockBean({
-    AccessTierServiceImpl.class,
-    WorkspaceAdminServiceImpl.class,
+    AccessTierService.class,
```

Comment:
because they're mocks we only need to include the interfaces here, not the Impls